src/providers/LocalizationProvider.tsx

```diff
@@ -23,9 +24,14 @@ export default function LocalizationProvider({
 }: LocalizationProviderProps): JSX.Element | null {
   const [timeZones, setTimeZones] = useState<TimeZoneDescription[]>([]);
 
+  useEffect(() => {
+    axios.defaults.headers = { ...axios.defaults.headers, 'Accept-Language': locale };
```

Comment:
Not sure if there's a more idiomatic way to do this.

Reply:
This is fine. Should we clear the header value if locale is not set?
Another way is using an [interceptor](https://axios-http.com/docs/interceptors).
No preference here, just fyi.
I should think about abstracting the network functionality into an Http service at some point... just an aside.

Reply:
There's no way to unset the locale -- it defaults to English (in `AppBootstrap`) and can only be updated to other nonempty values.

Reply:
Got it, at some point when we read the locale from the user (to override the default locale).. guessing we will ignore unset locales.

Reply:
Yep, and in fact I just submitted a PR to do exactly that: #1092 .